tests/asgi/test_ws.py

```diff
@@ -1087,7 +1087,8 @@ async def on_websocket(self, req, ws):
 
 async def test_ws_simulator_collect_edge_cases(conductor):
     class Resource:
-        pass
+        async def on_websocket(self, req, ws):
```

Comment:
Maybe adding an unrelated method (such as `on_delete` or whatever) would match the original intent of this test case better? (I.e., not having any `on_websocket()` responder.)